server/controllers/articles.controller.js

```diff
@@ -1,16 +1,36 @@
-const { fetchArticle, fetchArticles } = require("../models/articles.model");
+const {
+  fetchArticle,
+  fetchArticles,
+  updateArticlebyArticleId,
+} = require("../models/articles.model");
 
 exports.getArticles = (req, res, next) => {
-  return fetchArticles().then((rows) => {
+  fetchArticles().then((rows) => {
     res.status(200).send({ articles: rows });
   });
 };
 
 exports.getArticle = (req, res, next) => {
   const { article_id } = req.params;
-  return fetchArticle(article_id)
+  fetchArticle(article_id)
     .then((article) => {
-      res.status(200).send({ articles: article });
+      res.status(200).send({ article });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.patchArticlebyArticleId = (req, res, next) => {
+  const { inc_votes } = req.body;
+  const { article_id } = req.params;
+
+  fetchArticle(article_id)
+    .then(() => {
+      return updateArticlebyArticleId(article_id, inc_votes);
```

Comment:
This works the way you've done it, but it isn't actually necessary to call the fetchArticle model here. You can throw a Promise.reject from your model if the article doesn't exist. 
Remember, we only want to be checking if the article exists in a separate model if we want to send back an empty array. For this endpoint however, if the article doesn't exist, we're always going to throw an error 👍